### PR TITLE
Change Bluemix to IBM Cloud

### PR DESCRIPTION
**Pull request by dolph**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -309,7 +309,7 @@ The user names and passwords can be modified in the /server/boot/init-access.js
 
 # Privacy Notice
 
-The following information is sent to a [Deployment Tracker](https://github.com/IBM-Bluemix/cf-deployment-tracker-service) service on each deployment:
+The following information is sent to a [Deployment Tracker](https://github.com/IBM-IBM Cloud/cf-deployment-tracker-service) service on each deployment:
```
Comment (dolph) [inline README.md]: This is going to 404.

File: README.md [hunk 2]
```diff
@@ -322,7 +322,7 @@ The following information is sent to a [Deployment Tracker](https://github.com/I
 * Labels of bound services
 * Number of instances for each bound service and associated plan information
 
-This data is collected from the `package.json` file in the sample application and the `VCAP_APPLICATION` and `VCAP_SERVICES` environment variables in IBM Bluemix and other Cloud Foundry platforms. This data is used by IBM to track metrics around deployments of sample applications to IBM Bluemix to measure the usefulness of our examples, so that we can continuously improve the content we offer to you. Only deployments of sample applications that include code to ping the Deployment Tracker service will be tracked.
+This data is collected from the `package.json` file in the sample application and the `VCAP_APPLICATION` and `VCAP_SERVICES` environment variables in IBM Cloud and other Cloud Foundry platforms. This data is used by IBM to track metrics around deployments of sample applications to IBM IBM Cloud to measure the usefulness of our examples, so that we can continuously improve the content we offer to you. Only deployments of sample applications that include code to ping the Deployment Tracker service will be tracked.
```
Comment (markstur) [inline README.md]: "IBM IBM" is redundant (or twice as nice?)

File: README.md [hunk 3]
```diff
@@ -39,24 +39,24 @@ This Accelerator requires 2 services to be created as well as WKS 2.0 for traini
...
-## Setting up Bluemix
+## Setting up IBM Cloud
 
-[![Deploy to Bluemix](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/IBM/voice-of-the-customer)
+[![Deploy to IBM Cloud](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/IBM/voice-of-the-customer)
 
-1. If you do not already have a Bluemix account, [sign up here](https://console.ng.bluemix.net/registration).
-2. Log into Bluemix with your own credentials.
-3. Create a new application by clicking on the Create App button on the Bluemix Dashboard.
+1. If you do not already have a IBM Cloud account, [sign up here](https://console.ng.bluemix.net/registration).
```
Comment (markstur) [inline README.md]: s/a IBM/an IBM/

**Comment by markstur** [comment]
automatically generated fixes have issues.  Closing.  Will fix manually.

**Pull request closed without merging**